### PR TITLE
Session8 circle class and tests

### PR DESCRIPTION
**Pull request by Douglas-Klos**
Everything appears to be working

### CODE REVIEW

File: students/douglas_klos/extra/numerals/numerals.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    @arabic.setter
+    def arabic(self, value):
+        self._arabic = value
+
+    @property
+    def roman(self):
+        """ Returns the roman numeral for self._arabic """
+        number = self._arabic
+        roman_numeral = ''
+        while number >= 1000:
```
Comment (mdgregor) [inline students/douglas_klos/extra/numerals/numerals.py]: A lot of duplication here, you can tighten it up a bit by:
```
        roman_conversions = [
            ('M', 1000),
            ('CM', 900),
            ('D', 500),
            ('CD', 400),
            ('C', 100),
            ('XC', 90),
            ('L', 50),
            ('XL', 40),
            ('X', 10),
            ('IX', 9),
            ('V', 5),
            ('IV', 4),
            ('I', 1)
        ]

        for value in roman_conversions:
            while number >= value[1]:
                roman_numeral += value[0]
                number -= value[1]
```

File: students/douglas_klos/session8/examples/properties_example.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+NOTE: if your getters and setters are this simple: don't do this!
+
+"""
+
+
+class C:
+    def __init__(self):
+        self._x = None
+    @property
+    def x(self):
```
Comment (mdgregor) [inline students/douglas_klos/session8/examples/properties_example.py]: don't forget to leave a blank line between functions.

File: students/douglas_klos/session9/examples/object_canvas.py [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+        self.diameter = diameter
+        super().__init__(**kwargs)
+
+    def draw(self, drawer):
+        """
+        Draw the object
+        :param drawer: PIL.ImageDraw object to draw to
+        """
+        r = self.diameter // 2
+        c = self.center
```
Comment (mdgregor) [inline students/douglas_klos/session9/examples/object_canvas.py]: no single letter variable if you can.  be descriptive.

Comment (Douglas-Klos) [reply students/douglas_klos/session9/examples/object_canvas.py]: I didn't write the things in the example folder, they were available for download on the https://uwpce-pythoncert.github.io/PythonCertDevel/exercises/index.html site.

**Comment by Douglas-Klos** [comment]
Yeah thanks :)  I knew there was a better way to handle those roman numerals.  Yeah, was a bit bored and found several extra little assignments on the class github so thought I'd give them a try, learn as much as I can.

File: students/douglas_klos/session9/examples/object_canvas.py (hunk 3 above)
Comment (mdgregor) [reply students/douglas_klos/session9/examples/object_canvas.py]: haha, sorry.  many times I mix up what belongs to what and start doing PR comments on instructor code :)

**Pull request merged**